modules/node_modules/@ncigdc/components/Aggregations/ExactMatchFacet.js

```diff
@@ -82,9 +82,10 @@ const ExactMatchFacet = compose(
                     ], false),
                   }}
                   dark={!!inputValue}
-                  onClick={() => setInputValue('')}
```

Comment:
Is this because the link wasn't calling the onClick? I wonder if it makes more sense to get Link to call onClicks passed in?